templates/templates/global-nav.html

```diff
@@ -18,78 +18,70 @@
       <li class="global-nav__link--dropdown p-inline-list__item">
         <a class="global-nav__link-anchor" href="#canonical-products">Products</a>
       </li>
-      <li class="global-nav__link p-inline-list__item">
-        <a class="global-nav__link-anchor" href="https://www.canonical.com">Company</a>
-      </li>
-      <li class="global-nav__link p-inline-list__item">
-        <a class="global-nav__link-anchor" href="#">Login</a>
-      </li>
     </ul>
   </div>
   <div class="global-nav__dropdown-content u-hide">
-    <div class="p-strip--dark is-shallow">
+    <div class="p-strip--dark is-shallow u-no-padding--bottom">
       <div class="row">
         <ul class="p-matrix">
           <li class="p-matrix__item">
-            <a class="p-matrix__img" href="https://www.ubuntu.com">
-             <img src="{{ ASSET_SERVER_URL }}c5cb0f8e-picto-ubuntu.svg" alt="icon">
-              <div class="p-matrix__content">
-                <h4 class="p-matrix__title"><a class="p-link" href="https://www.ubuntu.com/">Ubuntu&nbsp;&rsaquo;</a></h4>
-                <p class="p-matrix__desc">The world’s most popular Linux for servers, desktops and things, with enterprise support and enhanced security by Canonical</p>
-              </div>
-            </a>
+            <img class="p-logomark" src="{{ ASSET_SERVER_URL }}c5cb0f8e-picto-ubuntu.svg" alt="icon">
```

Comment:
Logomark?

Reply:
That's the class used in the enterprise menu - it was pre-existing so reused it here as the same layout was required.